elements/pfe-codeblock/DISCOVERY.md

```diff
@@ -0,0 +1,40 @@
+# Project name
```

Comment:
This can be deleted since it wasn't used

Reply:
@castastrophe, Delete the whole file?